### PR TITLE
Fix/env warning

### PR DESCRIPTION
**Pull request by DAsfLeanIX**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "npx @vercel/ncc build index.js"
```
Comment (perbernhardt) [inline package.json:8]: npx! Nice :-)

File: update-dist.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+npm i -g @vercel/ncc
```
Comment (perbernhardt) [inline update-dist.sh:1]: I think we can remove update-dist.sh completely as you use the package/scripts option to define the call.

**Review by perbernhardt — CHANGES_REQUESTED** [review]
... and please remove the "node_modules" from git completely and also add it to the .gitignore. Thanks!

File: update-dist.sh (hunk 2 above)
Comment (DAsfLeanIX) [reply update-dist.sh]: It's removed now

**Comment by DAsfLeanIX** [comment]
> ... and please remove the "node_modules" from git completely and also add it to the .gitignore. Thanks!

Removed the node_modules and added the folder to gitignore!

**Review by perbernhardt — APPROVED** [review]

**Pull request closed without merging**